README.md

```diff
@@ -33,6 +33,18 @@ Returns a continuous, hex-prefixed hex value for the signature, suitable for inc
 
 Takes an address of either upper or lower case, with or without a hex prefix, and returns an all-lowercase, hex-prefixed address, suitable for submitting to an ethereum provider.
 
+### encrypt(msgParams)
+
+msgParams should have a `data` key that is hex-encoded data unencrypted, and a `sig` key that is hex-encoded and signed.
+
+Returns value of `data` encrypted with public key used to sign `sig`
```

Comment:
Is a hex encoded sig the ideal input, or would an object with r, s, and v values be easier to manage? I’m on a phone, so not asking rhetorically, maybe worth comparing to other sig methods.